apps/mull-ui/src/app/pages/home/my-events/my-events.spec.tsx

```diff
@@ -27,8 +27,8 @@ describe('myEventsPage', () => {
       const mocks: MockedResponse[] = [
         {
           request: {
-            query: GetUsersEventsDocument,
-            variables: { UserId: 1 },
+            query: OwnedEventsDocument,
```

Comment:
I'm not familar with graphql codegen yet, why does it add "Document" at the end?

Reply:
That's the raw query string, which is what we need to put here.